### PR TITLE
tomsanch doc-update

### PR DESCRIPTION
**Pull request by tomsanch**
Definitely needs a thorough update in coming months, but this is a good start.

### CODE REVIEW

File: pages/about.vue [hunk 1]
```diff
@@ -283,13 +283,9 @@
...
-        overlord that commands them to use their web browser to navigate to a
-        certain website and take a snapshot or copy some text that they must
-        report. Indeed, the miners are just computers running a software
-        (Witnet-rust) that automatically receive and execute a series of tasks
-        without the owner of the computer having to actively do anything else
-        than installing it.
+        front of a computer, fulfilling assignments. Instead, 
+        the miners are just computers running the witnet-rust software
+        that automatically receive and execute a series of tasks
       </p>
```
Comment (clbartoli) [inline pages/about.vue:289]:
```suggestion
        front of a computer, fulfilling assignments. Instead, the miners are
        just computers running the witnet-rust software that automatically
        receive and execute a series of tasks
      </p>
```

File: pages/about.vue [hunk 2]
```diff
@@ -805,30 +801,30 @@
...
-          chain). These will be assigned to early contributors (participants in
+          30% will be minted in the Genesis Block: the first block in the
+          chain. These will be assigned to participants in
           the pre-sale conducted in 2017; the Witnet Foundation; and those who
           helped to build Stampery<a class="link" href="#footnote2">
             <sup>2</sup></a
           >)
         </li>
       </ul>
       <p>
```
Comment (clbartoli) [inline pages/about.vue:812]:
```suggestion
          30% will be minted in the Genesis Block: the first block in the chain.
          These will be assigned to participants in the pre-sale conducted in
          2017; the Witnet Foundation; and those who helped to build Stampery<a
            class="link"
            href="#footnote2"
          >
            <sup>2</sup></a
          >)
        </li>
      </ul>
      <p>
```

File: pages/about.vue [hunk 3]
```diff
@@ -805,30 +801,30 @@
...
-        The mined tokens will be released in the following way, subject to
-        change: <br />
-        The number of WITs generated per block starts at 1000 and is set to
-        decrease geometrically, with a 50% reduction every 875,000 blocks, or
-        approximately every 2.5 years. Each of these periodic reductions is
+        The mining schedule for tokens is as follows: <br />
+        The number of WITs generated per block starts at 500 and is set to
+        decrease geometrically, with a 50% reduction every 1,750,000 blocks, or
+        approximately every 5 years. Each of these periodic reductions is
         known as "halving". As a result, the total number of WITs created by the
```
Comment (clbartoli) [inline pages/about.vue:817]:
```suggestion
        approximately every 5 years. Each of these periodic reductions is known
        as "halving". As a result, the total number of WITs created by the
```

File: pages/about.vue [hunk 4]
```diff
@@ -805,30 +801,30 @@
...
         issuance mechanism will never exceed 2,500,000,000 minus the preassigned
-        30%.
+        30%. <br />
+        Note: these figures are open to debate within the community, and subject to change.
```
Comment (clbartoli) [inline pages/about.vue:820]:
```suggestion
        Note: these figures are open to debate within the community, and subject
        to change.
```

**Comment by mariocao** [comment]
@tomsanch any update on this PR?

**Pull request merged**